pytket/phir/phirgen.py

```diff
@@ -27,6 +27,8 @@
 
 logger = logging.getLogger(__name__)
 
+JsonDict = dict[str, Any]
+
 PHIR_HEADER: dict[str, Any] = {
```

Comment:
```suggestion
PHIR_HEADER: JsonDict = {
```

Another similar change is at line 364. You could also consider updating `phirgen_parallel.py` while you are at it.